src/sandstorm/web-session.capnp

```diff
@@ -172,6 +173,14 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
     encoding @2 :Text;  # Content-Encoding header (optional).
   }
 
+  struct PatchContent {
+    # TODO(apibump): Remove this and replace it with `PostContent` (renamed to `Content`).
```

Comment:
Since this is a newly-introduced struct, can you go ahead and use PostContent? It turns out it was a mistake to have separate structs for each method, so we're planning to merge everything into PostContent (and rename it). Some of the WebDAV methods are already using PostContent.